### ISSUE TITLE
Can't login via EasyRSS but can via browser

### ISSUE THREAD

**Issue by ghost** [FreshRSS/FreshRSS#1497]
Same error as in #962 except all iterations of the URL haven't worked for me. I've enabled the API and set the password to the same as web login. I'm using the Cloudron install

**Comment by Alkarex** [comment]
* What is the path of FreshRSS on your server?
* Please perform the tests by clicking on the link of:
![image](https://cloud.githubusercontent.com/assets/1008324/24933527/0f621e60-1f17-11e7-9d85-ed589ebf6c67.png)

* Please have a look at your HTTP logs to see whether the requests are coming from your client.
* Please check the API logs in `./data/users/_/log_api.txt`

**Comment by ghost** [comment]
I haven't got any link next to the API password box: 

![pic](http://i.imgur.com/WPhBMsx.png)

The other parts I'm not exactly sure where to look, I'm using Cloudron because I have basically near zero sysadmin skills. If you can provide the paths for the files I can take a look though.

**Comment by Alkarex** [comment]
Ah, sorry, the link is from a patch that is not in /dev yet... ( https://github.com/FreshRSS/FreshRSS/pull/1292 )
If the address of your FreshRSS contains a `/p/` , try to browse to `https://example.net/.../p/api/` and if it does not, try to browse to `https://freshrss.example.net/api/`.
Anyway, look at `./data/users/_/log_api.txt` in the FreshRSS directory. Let me know if you cannot locate it.
Maybe @gramakri could have some input.

**Comment by ghost** [comment]
It has a /p/, when I go to https://domain.com/p/api/ it redirected me to https://domain.com:8000/p/api/ and timed out. 

Yeah I'm not sure where to find the FreshRSS directory on Cloudron :-/

**Comment by gramakri** [comment]
Just tried this quickly.
* Enabled API access
* Visited /p/api/ on browser (trailing slash important!). The full server configuration says 'Not found' but 'Check Partial server configuration' succeeds with PASS. Per [this comment](https://github.com/FreshRSS/FreshRSS/issues/443#issuecomment-36666133) this is not a problem since I am going to use EasyRSS.
![image](https://cloud.githubusercontent.com/assets/82041/24981253/c6e37308-1f90-11e7-8913-5bb694638a32.png)
*  Installed EasyRSS.
* Cannot Login with `https://<cloudron domain>/p/api/greader.php as the URL`.
* It seems that API password is not set by the Cloudron app. I couldn't figure how to set the API password of an existing user. So, I created a new user using the CLI as `cli/create-user.php --user girish --password password --api-password apipassword`.
* Logged in with above credentials in EasyRSS and it worked!

Long story short, the api password is not set in the Cloudron app. How can I set the API password of an existing user? Alternately, can FreshRSS allow me to set the API password in the web interface?

cc @dswd

**Comment by Alkarex** [comment]
Good to see some progress :-)
Yes, you can set the API password via the Web interface:

![image](https://cloud.githubusercontent.com/assets/1008324/24981512/5d064a3a-1fdd-11e7-994b-7eaa70323241.png)

**Comment by gramakri** [comment]
@Alkarex Is that a new feature in dev or do I have to enable this in some way? I don't see this on the Cloudron app.

![image](https://cloud.githubusercontent.com/assets/82041/24983581/cf458ba6-1f9c-11e7-8cd3-c31e1d9ab63f.png)

**Comment by Alkarex** [comment]
Hum, no, it should be there in your version.
Try to see whether the API option is enabled:

![image](https://cloud.githubusercontent.com/assets/1008324/24983717/56c5ef0c-1fe9-11e7-969b-7f3d6e0c3a63.png)

Related to https://github.com/FreshRSS/FreshRSS/issues/1432

**Comment by dswd** [comment]
Hi all, it is funny that I tried to use FreshRSS with EasyRSS just two days ago without knowing about this thread. I can confirm that access via EasyRSS works when API access is enabled (as admin user) and an API password is set for the user.

**Comment by dswd** [comment]
If there is anything I can do in the Cloudron packaging to make this easier for users, please tell me.

**Comment by gramakri** [comment]
You are right, it's right there, I am able to set the API password. Maybe I disabled API by mistake.

@Alkarex @dswd I guess we can close this bug?

@LeapingDragon maybe you can hop on chat.cloudron.io and we can help you out if you are still facing the problem.
